Source/WebCore/css/CSSProperties.json

```diff
@@ -5067,8 +5067,8 @@
             "codegen-properties": {
                 "custom": "Value",
                 "initial": "initialOneLength",
-                "converter": "Length",
-                "parser-grammar": "<length-percentage svg>"
+                "converter": "LengthAllowingNumber",
+                "parser-grammar": "<length-percentage [0,inf]> | <number [0,inf]>"
```

Comment:
Since this matches the SVG2 draft definition, perhaps we should update the specification link to that (instead of https://drafts.fxtf.org/fill-stroke-3/#propdef-stroke-width, which is currently defined to `<length-percentage [0,inf]>#`):

https://svgwg.org/svg2-draft/painting.html#StrokeWidth